kuma/authkeys/templates/authkeys/list.html

```diff
@@ -22,14 +24,21 @@
                 <a class="button"
                    title="{{ _('View key usage history') }}"
                    href="{{ url('authkeys.history', pk=key.pk) }}">{{ _('History ({count})') | f(count=key.history.count()) }}</a>
-                <a class="button transparent only-icon"
-                   href="{{ url('authkeys.delete', pk=key.pk) }}">
-                   <span>{{ _('Delete key') }}</span><i aria-hidden="true" class="icon-trash"></i>
-                </a>
             </span>
 
         </li>
         {% endfor %}
     </table>
 {% endif %}
+
+{% else %}
+<p>{% trans %}
+  You need a special permission to use API keys.
+  You may request permission by <a href="https://bugzilla.mozilla.org/enter_bug.cgi?format=__standard__&product=Mozilla Developer Network&component=User management">filing a bug</a>.
```

Comment:
When I moved the url up into the `trans` declaration it was automatically url-encoded by jinja. :gift: